### DISCUSSION TITLE
My pull request are accepted but on the hacktoberfest page it is showing updated.

### DISCUSSION THREAD

**Discussion by Vaishnavi-More** [YoshithaRathnayake/NEW-YOSHITHA-WEBSITE#118] [Q&A]
Hey!
My request are merged but I am unable to see my progress rate on hacktoberfest page. It is showing 4 waiting. Do you have any idea how much time it take to update?

**Comment by YoshithaRathnayake** [comment]
> Hey! My request are merged but I am unable to see my progress rate on hacktoberfest page. It is showing 4 waiting. Do you have any idea how much time it take to update?

Yes it's taking 7 days to accept I did it correctly but I don't know about the other one that merged your two PRs @Vaishnavi-More

**Reply by Vaishnavi-More** [reply]
Okay fine!
Thank you for clearing my doubt!

**Reply by YoshithaRathnayake** [reply]
> Okay fine! Thank you for clearing my doubt!

can you mark my answer as correct pls @Vaishnavi-More

**Reply by Vaishnavi-More** [reply]
Okay!

**Reply by YoshithaRathnayake** [reply]
Did you mark my answer as correct pls do it for me @Vaishnavi-More

**Reply by YoshithaRathnayake** [reply]
Thanks @Vaishnavi-More
